frappe/desk/doctype/todo/todo.py

```diff
@@ -97,3 +97,11 @@ def has_permission(doc, user):
 		return True
 	else:
 		return doc.owner==user or doc.assigned_by==user
+
+@frappe.whitelist()
+def new_todo(description):
+	frappe.get_doc({
+		'doctype': 'ToDo',
+		'description': description
+	}).insert()
+	frappe.local.flags.commit = True
```

Comment:
You don't need to use `frappe.local.flags.commit` if you use `type: "POST"` in `frappe.call`